packit_service/models.py

```diff
@@ -225,7 +225,7 @@ def get_project_branches(
             branches = (
                 session.query(GitBranchModel)
                 .filter_by(project_id=project.id)
-                .order_by(desc(GitBranchModel.id))
+                .order_by(GitBranchModel.id)
```

Comment:
Was this intentional?

Reply:
Yes because master branch is more likely to have builds (ex. packit-master) than other branches. And it almost always has a lower id than the other branches. So I thought lets just show them in ascending order so that master is at the top.